ci/configure-bazel.sh

```diff
@@ -64,8 +68,8 @@ if is_windows; then
   # To avoid exceeding the maximum path limit on Windows we limit the suffix to
   # three characters.
   echo "Working directory: $PWD"
-  SUFFIX="$(echo $PWD | md5sum)"
-  SUFFIX="${SUFFIX:0:2}"
+  SUFFIX="$(echo $PWD $RULES_HASKELL_REV | md5sum)"
```

Comment:
Note that we have openssl in dev-env, so you could use:
```suggestion
  SUFFIX="$(echo $PWD $RULES_HASKELL_REV | openssl dgst -md5 -binary | openssl enc -base64)"
```
to get an md5 as a base64-encoded string, rather than hex-encoded. Wider alphabet should mean fewer chances of collision.